docs/release_notes/v1.0.0-rc.2.md

```diff
@@ -0,0 +1,533 @@
+dapr-sentry-84565c747b-ldnsf             1/1     Running   0          29s
+dapr-sidecar-injector-68f868668f-6xnbt   1/1     Running   0          41s
+dapr-sidecar-injector-68f868668f-j7jcq   1/1     Running   0          29s
+dapr-sidecar-injector-68f868668f-ltxq4   1/1     Running   0          36s
+```
+
+
+Restart your application deployments to update the Dapr runtime.
+```sh
+kubectl rollout restart deploy/<DEPLOYMENT-NAME>
+```
+
+All done!
+
+#### Starting fresh install on a cluster 
+
+You can use Helm 3 to install Dapr:
+```sh
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+kubectl create namespace dapr-system
+helm install dapr dapr/dapr --version 1.0.0-rc.2 --namespace dapr-system
+```
+
+Alternatively, you can use the latest version of CLI:
+
+```
+dapr init --runtime-version=1.0.0-rc.2 --kubernetes
+```
+
+If you previously installed Dapr on your Kubernetes cluster using the Dapr CLI, run:
+
+*Note: Make sure you're uninstalling with your existing CLI version*
+
+```bash
+dapr uninstall --kubernetes
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+##### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+$ kubectl get pods -n dapr-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+dapr-dashboard-85cc6c9f7d-hzk69         1/1     Running   0          13m
+dapr-operator-84985cb985-h2zdv          1/1     Running   0          13m
+dapr-placement-server-0                 1/1     Running   0          13m
+dapr-sentry-5974c7cb6d-9bnqd            1/1     Running   0          13m
+dapr-sidecar-injector-5b5bb9454-sfmmx   1/1     Running   0          13m
+```
+
+After Dapr 1.0.0-rc.2 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Dapr Runtime
+
+- **FIXED** Pubsub API will now return 403 for forbidden pubsub topics. [2323](https://github.com/dapr/dapr/issues/2323)
+
+- **MOVED** Zipkin trace configuration from Zipkin exporter into Dapr core [2521](https://github.com/dapr/dapr/issues/2521)
+
+### Dapr CLI
+
+- **MADE** shorthands across commands consistent [530](https://github.com/dapr/cli/issues/530)
+
+- **UPDATED** Default standalone Configuration to not use deprecated Zipkin exporter component. [534](https://github.com/dapr/cli/issues/534)
+
+- **ADDED** `--publish-app-id` to the publish command [536](https://github.com/dapr/cli/issues/536)
+
+- **REMOVED** unused and unsupported `--image` flag from `run` command. [537](https://github.com/dapr/cli/issues/537)
+
+- **REMOVED** `redis-host` flag from `dapr init` [540](https://github.com/dapr/cli/issues/540)
+
+### .NET SDK
```

Comment:
It'd be good to include a link to the mitigation guide here:

https://github.com/dapr/dotnet-sdk/issues/505

Reply:
We do that on the top. This is just a simple list of issues closed.

Reply:
Yes, and I'd like to keep it simple too. From discussions with @msfussell he feels it'd be good to reference the mitigations here so people have to spend less time digging. 

So, we don't need the whole mitigation here but could do something simple like:

Note: See [upgrade] for mitigation instructions.

Reply:
I can add this.